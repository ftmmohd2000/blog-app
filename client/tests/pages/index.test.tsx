import React from "react";
import { ShallowWrapper, shallow } from "enzyme";
import Home from "../../pages";

let wrapper: ShallowWrapper;

beforeEach(() => {
  wrapper = shallow(<Home />);
});

test("should render Home page correctly", () => {
  expect(wrapper).toMatchSnapshot();
});
