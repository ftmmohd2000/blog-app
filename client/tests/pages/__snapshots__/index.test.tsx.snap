// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home page correctly 1`] = `
<Component
  pageTitle="bruh"
>
  <p />
</Component>
`;
